oio/cli/directory/directory.py

```diff
@@ -100,7 +106,9 @@ class DirectoryInit(DirectoryCmd):
     def get_parser(self, prog_name):
         parser = super(DirectoryInit, self).get_parser(prog_name)
         parser.add_argument(
-            '--no-rdir', action='store_true', help='Deprecated')
+            '--level', metavar='<0,1,2,3>', dest='level',
```

Comment:
In the C language load balancer, the *level* is 0 for the rightmost location token. This can create confusion. Could we talk about _distance_ or _remoteness_ instead?

Reply:
Now more precise, we expect a self-explanatory token ("site", "rack", "host","volume").